pkgs/os-specific/linux/vendor-reset/default.nix

```diff
@@ -18,6 +18,12 @@ stdenv.mkDerivation rec {
       url = "https://github.com/gnif/vendor-reset/commit/5bbffcd6fee5348e8808bdbfcb5b21d455b02f55.patch";
       sha256 = "sha256-L1QxVpcZAVYiaMFCBfL2EJgeMyOR8sDa1UqF1QB3bns=";
     })
+    # Fix build with Linux 6.8+
+    # https://github.com/gnif/vendor-reset/pull/77
+    (fetchpatch {
```

Comment:
PRs can be force pushed at any time, which can make commits that are just on PR branches go away.  For longevity, it's best to vendor patches that haven't been merged yet (or just wait for the merge).

Reply:
Makes sense. I've vendored the patch.